scripts/build/docker/compile-template.js

```diff
@@ -31,11 +31,11 @@ const template = fs.readFileSync(path.join(currentDirectory, 'Dockerfile.templat
 _.each([
   {
     architecture: 'i686',
-    image: 'erwinchang/ubuntu-12.04-32bit-build'
+    image: 'erwinchang/ubuntu-14.04-32bit-build'
   },
   {
     architecture: 'x86_64',
-    image: 'ubuntu:12.04'
+    image: 'ubuntu:14.04'
```

Comment:
As you're probably aware, the reason that Etcher (for Linux) was being built on Ubuntu 12.04 was because that's the version that https://electronjs.org/docs/tutorial/support claims Electron (for x86 and amd64 Linux versions) was built on, and (at the time) Etcher was aiming for compatibilty with as many different Linux distros as possible (IIRC this effectively came down to the [glibc](https://github.com/resin-io/etcher/search?q=glibc&type=Issues) version used by that old Ubuntu 12.04 version).
But given that Ubutunu 12.04 is now EOL with no security updates, it probably makes sense to drop support for it (just like you did for WindowsXP). Ubuntu 14.04 LTS is supported until April 2019 https://www.ubuntu.com/info/release-end-of-life

IIRC some of the custom repos and packages that we had to add to these Docker scripts were purely because we were using such an old version of Ubuntu, so it might be worth revisiting those and seeing which ones can be removed now you're buidling on an Ubuntu 14.04 base?